src/containers/BooksList.js

```diff
@@ -1,21 +1,31 @@
 import React from 'react';
-import { useSelector } from 'react-redux';
+import { useSelector, useDispatch } from 'react-redux';
 import Book from '../components/Book';
+import { removeBook } from '../actions/index';
 
 function BooksList() {
+  const dispatch = useDispatch();
+
+  function handleRemoveBook(book) {
```

Comment:
### [Optional]
- It would be better if we use ES6 syntax instead of the function keyword. We are using `const` which is ES6 so it would be better if we follow with the consistency